subcommand/consul-sidecar/command.go

```diff
@@ -1,4 +1,4 @@
-package subcommand
+package consulsidecar
```

Comment:
Everything past this in the PR is a merge from feature-metrics which is the base of feature-tproxy.
I've rebased feature-tproxy off feature-metrics and pushed it but for some reason it's not showing up here.
Will try to sort it out before merging.